<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NNLDemo</class>
 <widget class="QMainWindow" name="NNLDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Demo for NNL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>301</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="bt_yes">
       <property name="text">
        <string>Yes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bt_no">
       <property name="text">
        <string>No</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>519</width>
      <height>149</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Your name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_user_name"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>582</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <widget class="QMenu" name="menuComponents">
    <property name="title">
     <string>Components</string>
    </property>
    <addaction name="actionNew_thing"/>
   </widget>
   <widget class="QMenu" name="menuValues">
    <property name="title">
     <string>Values</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuComponents"/>
   <addaction name="menuValues"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNew_thing">
   <property name="text">
    <string>New thing</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>NNLDemo</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>290</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
